config/pacbio_instrument_types.yml

```diff
@@ -4,6 +4,7 @@ default: &default
     name: Sequel IIe
     smrt_link_versions:
       - v11
+      - v12
```

Comment:
Maybe rename `sequel_iie` to `v12_sequel_iie` on line 3 for consistency?

Reply:
@StephenHulme Steve is off for a few weeks yet, so happy to discuss with @BenTopping ?

Reply:
I'm not sure on the purpose of the smrt_link_versions as v12 isn't an actual version, it feels like it should be `v12_revio` or `v12_sequel_iie` for the given system. Maybe the idea was to add validation to check the smrtlink version was valid for a given instrument?